Core/GraphicsInterops/Source/ExternalTexture_D3D11.cpp

```diff
@@ -0,0 +1,191 @@
+#include <Babylon/Graphics/Device.h>
+#include <Babylon/GraphicsInterops/ExternalTexture.h>
+#include <Babylon/Graphics/RendererType.h>
+#include <Babylon/Graphics/Texture.h>
+#include <Babylon/Graphics/DeviceContext.h>
+
+#include <napi/env.h>
+#include <napi/napi_pointer.h>
+
+namespace Babylon::Graphics
+{
+    //For refrence see renderer_d3d11.cpp line 203.
+    bgfx::TextureFormat::Enum NativeToBGFXImageFormat(DXGI_FORMAT format)
+    {
+        switch (format)
+        {
+            case DXGI_FORMAT_BC1_UNORM:
+            case DXGI_FORMAT_BC1_UNORM_SRGB:
+                return bgfx::TextureFormat::Enum::BC1;
```

Comment:
The srgb part needs to be translated also. They are not the same.

Reply:
Added support for SRGB textures on the latest commit.